### DISCUSSION TITLE
migrate milvus cluster mode from docker-compose to milvus cluster mode on kubernetes

### DISCUSSION THREAD

**Discussion by MayurDuduka** [milvus-io/milvus#27867] [Q&A and General discussion]
- **source:** Milvus version on docker-compose:  v2.3.1 (cluster mode)
- **destination:** Milvus version on Kubernetes:  v2.3.1 (cluster mode)

**We want to migrate all the collections to Kubernetes Milvus from Milvus docker-compose.**

Can someone please assist with the steps here?

Thanks

**Comment by yhmo** [comment]
The [milvus-backup](https://github.com/zilliztech/milvus-backup) tool is the recommended tool to backup/restore collections from a milvus to another.
Let's say your source Milvus is using MinIO service A as storage. The destination Milvus is using MinIO service B as storage.
The steps:
1. Use the milvus-backup tool to backup data files of a collection, and upload the data files to another path of service A.
2.  Manually copy(or use minio sdk to copy) the data files to a path of MinIO service B
3. Use the milvus-backup tool to restore the data files into server B
